snuba/admin/clickhouse/migration_groups.py

```diff
@@ -0,0 +1,8 @@
+from typing import Sequence, TypedDict
+
+from snuba.migrations.groups import MigrationGroup
+
+
+class MigrationGroupData(TypedDict):
```

Comment:
I would make this a dataclass/namedtuple instead of a TypedDict. I have a bit of a preference that way since TypedDict isn't quite as easy to type (you [can't do](https://mypy.readthedocs.io/en/stable/more_types.html#unions-of-typeddicts) `isinstance` checks).

It should JSON encode in exactly the same way.

Reply:
I would not even add this type here. It's only used briefly in an intermediate step of processing the API response.